README.md

```diff
@@ -21,7 +21,7 @@ metadata:
 spec:
   initContainers:
   - name: iam-waiter
-    image: docker.io/reactiveops/iam-waiter:0.1.0
```

Comment:
FYI - this doesn't seem to exist

Reply:
Woops, just seeing this now. Not sure if this is being used by anyone anymore and/or if it's worth spending any time on it, but it should be fairly straightforward to push an image somewhere for this if so. Dockerfile shouldn't have changed at all.